src/raftstore/store/store.rs

```diff
@@ -816,7 +816,6 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         assert!(!p.is_applying());
 
         let is_initialized = p.is_initialized();
-        let end_key = enc_end_key(p.region());
         if let Err(e) = p.destroy() {
```

Comment:
Yes, it is better to check initialization in destroy itself, but do we need set tombstone for the uninitialized peer?